### ISSUE TITLE
[BITV]: Replace <h2> element from 'Users' view to span

### ISSUE THREAD

**Issue by emoral435** [nextcloud/server#42554]
This issue messes up the the Heading hierarchic within the view of the 'Users' view. This is due to a NcVue component, so we will need to get this in the next NcVue package update before the next review

## Image for example

![Image](https://github.com/nextcloud/server/assets/110193237/54ff5621-bb32-40f8-bdbe-a05b764b005f)

**Comment by szaimen** [comment]
dependency update in stable28 is missing

**Comment by emoral435** [comment]
Will be good to check again once #43064 gets merged!
